JavaScript/news-assistant/README.md

```diff
@@ -0,0 +1,24 @@
+# News assistant
+- - - - - - - - -
+## Aim
+
+The aim of this script is to read out the top headlines of your favourite news category</br>
+Categories you can choose from are:</br>
+- business
+- entertainment
+- general
+- health
+- science
+- sports
+- technology
+
+## Requirements
+```npm install newsapi --save```</br>
+```npm install prompt-sync```</br>
+```npm install say```</br>
```

Comment:
![Capture](https://user-images.githubusercontent.com/50990883/98138359-054bd080-1ee9-11eb-92a5-7a4f1fa0f95d.PNG)

I am getting this error while installing package. Please check the commands

Reply:
Sir, it's ```prompt-sync```. 
Probably a typo

Reply:
Oh! Sorry, Tania. It was my mistake. Your code is running fine now!